docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -1233,7 +1233,7 @@ It is important that this error endpoint is a public resource to avoid the user
 == References
 
 * xref:security-oidc-configuration-properties-reference.adoc[OIDC configuration properties]
-* xref:security-openid-connect-providers.adoc[OpenID Connect providers]
+* xref:security-openid-connect-providers.adoc[Well Known OpenID Connect providers]
```

Comment:
IMHO, I think "well-known" is somewhat informal and think the term is either not needed or something like "Common" or even "Popular" might be better. 
If you prefer to keep "well known" here, just to note the capitalization/punctuation should be "Well-known OpenID Connect providers"

Reply:
@sheilamjones - Good point regarding _well-known_. +1 to popular or commonly used instead.

Reply:
I think "commonly-used" is a great suggestion as it is not subjective and still conveys the intended meaning. Would that sound okay to you @sberyozkin as an alternative to "well-known"?

Reply:
Hi @michelle-purcell @sheilamjones Thanks for the review, much appreciated, I think `Commonly used` was already added in one of the suggestions above but I'm not sure replacing `Well Known` everywhere with `Commonly used` everywhere would be very precise, i.e, everyone know `Twitter` but is it commonly used for the OIDC flow I'm not sure (same for `Apple` which requires a paid subscription).

`Well-Known` is actually a known term in OIDC, its configuration is on `.well-known` path segment

Reply:
Let me add `Configuring ...` if it is not added yet in some places